disnake/asset.py

```diff
@@ -118,6 +119,43 @@ async def save(
             with open(fp, "wb") as f:
                 return f.write(data)
 
+    async def to_file(self, *, spoiler: bool = False) -> File:
+        """|coro|
+
+        Converts the asset into a :class:`File` suitable for sending via
+        :meth:`abc.Messageable.send`.
+
+        .. versionadded:: 2.0
+
+        Parameters
+        -----------
+        spoiler: :class:`bool`
+            Whether the file is a spoiler.
+
+        Raises
+        ------
+        DiscordException
+            The asset does not have an associated state.
```

Comment:
```suggestion
            The asset does not have an associated state.
        InvalidArgument
            The asset is a unicode emoji.
```
might be good to add this as well (see `PartialEmoji`)